src/MeetingsSDKAdapter.test.js

```diff
@@ -809,8 +809,8 @@ describe('Meetings SDK Adapter', () => {
       meetingSDKAdapter
         .createMeeting(target)
         .pipe(flatMap(() => meetingSDKAdapter.getMeeting(meetingID)))
-        .subscribe((getMeeting) => {
-          expect(getMeeting).toMatchObject(meeting);
+        .toPromise().then((data) => {
+          expect(data).toMatchObject({...meeting, localAudio: null, localVideo: null});
```

Comment:
why these changes?

Reply:
I'm getting this error.
`
Test suite failed to run

    Call retries were exceeded

      at ChildProcessWorker.initialize
`